src/util/github.js

```diff
@@ -15,24 +15,24 @@ const getOctokit = () => {
 
 const getRepository = (owner, repo) => getOctokit().rest.repos.get({
     owner,
-    repo
+    repo,
 });
 
 const getRepositoryBranch = (owner, repo, branch) => getOctokit().rest.repos.getBranch({
     owner,
     repo,
-    branch
+    branch,
 });
 
 const getFileFromPages = async (owner, repo, file) => {
-    const { data: { html_url } } = await getOctokit().rest.repos.getPages({
+    const { data } = await getOctokit().rest.repos.getPages({
```

Comment:
You're right, only the handlers are tested, everything in that `util` folder isn't currently tested.  You can see the coverage results here: https://github.com/liatrio/semantic-release-helm/runs/5761419165?check_suite_focus=true#step:6:10.

But yeah, this change is okay, since I also changed this line: https://github.com/liatrio/semantic-release-helm/pull/11/files#diff-2b9691c756a4997d3facc425fb7e5e4335911b826b273559cc450c66d0a47c1bR28

Basically, we get a `data` object back, which has a property `html_url`.  I just changed how we accessed this property: instead of using a nested [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), I just accessed it directly.